docs/Concepts/Transactions/Transaction-Types.md

```diff
@@ -0,0 +1,58 @@
+---
+description: Description of the different transaction types
+---
+
+# Transaction types
+
+You can interact with the Hyperledger Besu JSON-RPC API using different transaction types (specified by
+the `transactionType` parameter).
+
+The following API objects use a unique format for each `transactionType`:
+
+- [Pending transaction object](../../Reference/API-Objects.md#pending-transaction-object)
+- [Transaction object](../../Reference/API-Objects.md#transaction-object)
+- [Transaction call object](../../Reference/API-Objects.md#transaction-call-object)
+- [Transaction receipt object](../../Reference/API-Objects.md#transaction-receipt-object)
+
+## `FRONTIER` transactions
+
+Transactions with type `FRONTIER` are *legacy transactions* that use the transaction format existing before typed
+transactions were introduced in [EIP-2718](https://eips.ethereum.org/EIPS/eip-2718).
+They contain the parameters `chainId`, `nonce`, `gasPrice`, `gasLimit`, `to`, `value`, `data`, `v`, `r`, and `s`.
+Legacy transactions are not compatible with [access lists](#access_list-transactions) or
+[EIP-1559 fee market changes](#eip1559-transactions).
+
+## `ACCESS_LIST` transactions
+
+Transactions with type `ACCESS_LIST` are transactions introduced in
+[EIP-2930](https://eips.ethereum.org/EIPS/eip-2930).
+They contain, along with the [legacy parameters](#frontier-transactions), an `accessList` parameter, which specifies an
+array of addresses and storage keys that the transaction plans to access (an *access list*).
+`ACCESS_LIST` transactions must specify an access list, and they are not compatible with
+[EIP-1559 fee market changes](#eip1559-transactions).
```

Comment:
similar thoughts about the use of 'compatibility' here.  Access list txns will continue to work, they just won't benefit from 1559 fee market changes.